examples/datetime_human_tests.nbt

```diff
@@ -32,7 +51,46 @@ assert((1e12 days        -> human) == "1_000_000_000_000 days")
 assert((1e15 days        -> human) == "1.0e+15 days")
 
 assert((1 ms             -> human) == "0.001 seconds")
+assert((12 ms            -> human) == "0.012 seconds")
+assert((342 ms           -> human) == "0.342 seconds")
+
 assert((1 µs             -> human) == "0.000001 seconds")
+assert((1245 µs          -> human) == "0.001245 seconds")
+assert((1236876 µs       -> human) == "1.236876 seconds")
+
 assert((1 ns             -> human) == "0.000000001 seconds")
 assert((1234 ns          -> human) == "0.000001234 seconds")
 assert((1s + 1234 ns     -> human) == "1.000001234 seconds")
+
+assert((-1 second        -> human) == "1 second ago")
+assert((-25 seconds      -> human) == "25 seconds ago")
+assert((-49.010 seconds  -> human) == "49.010 seconds ago")
+
+assert((-2.5 minute      -> human) == "2 minutes + 30 seconds ago")
+assert((-1 hour          -> human) == "1 hour ago")
+assert((-3 hour          -> human) == "3 hours ago")
+assert((-5.34 hour       -> human) == "5 hours + 20 minutes + 24 seconds ago")
+assert((-7.89 hour       -> human) == "7 hours + 53 minutes + 24 seconds ago")
+assert((-7.89 hour - 4 minutes -> human) == "7 hours + 57 minutes + 24 seconds ago")
+
+assert((-1 day           -> human) == "1 day ago")
+assert((-4.6 days        -> human) == "4 days + 14 hours + 23 minutes ago")
+assert((-9.2345 days     -> human) == "9 days + 5 hours + 37 minutes + 40.800 seconds ago")
+assert((-9.2345 days + 52 hours      -> human) == "7 days + 1 hour + 37 minutes + 40.800 seconds ago")
+
+assert((-2 weeks         -> human) == "14 days ago")
+assert((-4.5 weeks       -> human) == "31 days + 12 hours ago")
+assert((-20.5 weeks      -> human) == "143 days + 12 hours ago")
+
+assert((-2.5 months      -> human) == "76 days + 2 hours + 12 minutes + 39.385800 seconds ago")
+assert((-2.5 years       -> human) == "913 days + 2 hours + 31 minutes + 52.629600003 seconds ago")
+assert((-10.5 decades    -> human) == "38350 days ago")
+
+assert((-25 ms           -> human) == "0.025 seconds ago")
+assert((-6400 ms         -> human) == "6.400 seconds ago")
+
+assert((-2500 µs         -> human) == "0.002500 seconds ago")
+assert((-40525 µs        -> human) == "0.040525 seconds ago")
+
+assert((-5147 ns         -> human) == "0.000005147 seconds ago")
+assert((-23s + 1234 ns   -> human) == "22.999998766 seconds ago")
```

Comment:
The logic for "… ago" is relatively simple, right? In that case, we don't need to repeat all these tests here.

Reply:
> True but does protect against changes if someone switches the logic in some way in the future, and with the test base's overall speed I'd say it doesn't hurt anyone to keep these in

Speed is not the matter, but humans are going to read those tests and wonder what they are for. I would really prefer if we strip down the negative-time-duration tests to one or two.